alpine/3.12/8.0/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # syntax = docker/dockerfile:1.0.2-experimental
-ARG SPRYKER_PHP_VERSION=7.3.18
+ARG SPRYKER_PHP_VERSION=8.0.5
 
-FROM php:${SPRYKER_PHP_VERSION}-fpm-alpine3.10
+FROM php:${SPRYKER_PHP_VERSION}-fpm-alpine3.12
 
 ARG COMPOSER_VERSION=2.0.12
-ARG BLACKFIRE_VERSION=1.55.0
-ARG NEWRELIC_VERSION=9.16.0.295
-ARG TIDEWAYS_VERSION=5.3.14
+ARG BLACKFIRE_VERSION=1.49.0
```

Comment:
Hey, just a short question, is there a specific reason to downgrade Blackfire instead of upgrading to version 2? Is version 2 not compatible with PHP 8 ?

Reply:
Hey, thank you for your comment, Blackfire is updated to the latest stable version - (1.57).
Blackfire v2 support is scheduled and will be investigated soon.